caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php

```diff
@@ -1023,7 +1031,7 @@ public function pricing_metabox()
             // $this->_adminpage_obj->get_cpt_model_obj()->ID() > 0 ? ' ee-collapsible-closed' : ' ee-collapsible-open'
         );
         $timezone = $event instanceof EE_Event ? $event->timezone_string() : null;
-        do_action('AHEE_log', __FILE__, __FUNCTION__, '');
+        // do_action('AHEE_log', __FILE__, __FUNCTION__, '');
```

Comment:
I don't see the purpose of commenting this out, just get rid of it.

Reply:
right... ya I don't know why I do that : \